presto-main/src/test/java/com/facebook/presto/execution/TestPlannerWarnings.java

```diff
@@ -127,7 +127,7 @@ private static Plan createPlan(LocalQueryRunner queryRunner, Session session, St
                 queryRunner.getCostCalculator(),
                 ImmutableSet.copyOf(rules));
 
-        return queryRunner.createPlan(session, sql, ImmutableList.of(optimizer), LogicalPlanner.Stage.OPTIMIZED_AND_VALIDATED, warningCollector);
+        return queryRunner.createPlan(session, sql, ImmutableList.of(optimizer, queryRunner.translateExpressions()), LogicalPlanner.Stage.OPTIMIZED_AND_VALIDATED, warningCollector);
```

Comment:
We needed this since we now assumes everything shouldn't contain `OriginalExpression`. Otherwise we will have validation failure.

Reply:
Does this need to be a separate commit? If yes, is it before the other one?

Reply:
> Does this need to be a separate commit? If yes, is it before the other one?

You're right, let me change that. Thanks!